examples/vanilla/android/app/src/main/java/com/example/MainApplication.java

```diff
@@ -51,11 +57,22 @@ public ReactNativeHost getReactNativeHost() {
 
   @Override
   public void onCreate() {
+    RNPerformance.getInstance().mark("onCreateStart");
     super.onCreate();
     // If you opted-in for the New Architecture, we enable the TurboModule system
     ReactFeatureFlags.useTurboModules = BuildConfig.IS_NEW_ARCHITECTURE_ENABLED;
     SoLoader.init(this, /* native exopackage */ false);
     initializeFlipper(this, getReactNativeHost().getReactInstanceManager());
+    RNPerformance.getInstance().mark("onCreateEnd");
+    Bundle detail = new Bundle();
+    detail.putString("unit", "ms");
+    RNPerformance.getInstance().metric("bundleSize", 100, detail);
```

Comment:
The example doesn't really make sense (`bundleSize` with `ms` as unit). Maybe `byte` instead?